driver-core/src/main/com/mongodb/client/model/expressions/DateExpression.java

```diff
@@ -19,22 +19,117 @@
 import java.util.function.Function;
 
 /**
- * Expresses a date value.
+ * An instantaneous date and time. Tracks a UTC datetime, the number of
+ * milliseconds since the Unix epoch. Does not track the timezone.
  */
 public interface DateExpression extends Expression {
```

Comment:
I don't understand this sentence and it is not clear whether it adds any information to what follows after. What does it add?

Reply:
Which specific sentence of the 3 are you referring to? (There is some redundancy there, intended to stress and make certain implications clear that might not be immediately obvious to everyone.)

Reply:
Terrible, sorry) I was talking about _"An instantaneous date and time"_. It's true that people say something like "instantaneous coordinate" to emphasize that they are talking about the coordinate value at a specific time (though to me even that seems tautologous). However, when we talk about time, I can't understand understand what "instantaneous date/time" is - "the time value at a specific time" does not make sense.

Maybe we could just remove the first sentence?